### PR TITLE
Add continuous space tutorial

### PR DESCRIPTION
**Pull request by gpescia**
Tutorial on continuous space including Harmonic oscillator in 3D and gaussian cores in 1D.

### CODE REVIEW

File: docs/tutorials/gs_continuous_space.ipynb [hunk 1]
```diff
@@ -0,0 +1,1210 @@
+{
```
Comment (PhilipVinc) [inline docs/tutorials/gs_continuous_space.ipynb:1]: <p>Can you make the minimum version &gt;=3.5?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (PhilipVinc) [inline docs/tutorials/gs_continuous_space.ipynb:1]: <p>It seems you were running this with version 3.4.3.something. </p><p>Can you just say that this tutorial is valid for 3.5 (which still, is relatively old by now) to be sure?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (PhilipVinc) [inline docs/tutorials/gs_continuous_space.ipynb:1]: <p>I think we can cut this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (PhilipVinc) [inline docs/tutorials/gs_continuous_space.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #11.        distances = distances[...,idxs[0],idxs[1],:]
</pre><p>you already defined this above, so maybe you can reuse the definition from before?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (PhilipVinc) [inline docs/tutorials/gs_continuous_space.ipynb:1]: <p>to do or to remove?</p><p> </p><p>by the way the tutorial is well done, so if you add something more as well written as the above we would be very happy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

**Comment by gcarleo** [comment]
thank you @gpescia this looks great !

File: docs/tutorials/gs_continuous_space.ipynb (hunk 1 above)
Comment (gcarleo) [reply docs/tutorials/gs_continuous_space.ipynb:1]: <p>yes agreed, maybe actually Helium could be a separate tutorial, where we show also the condensate etc, we can ask David to do it later on </p>

Comment (gcarleo) [inline docs/tutorials/gs_continuous_space.ipynb:1]: <p>maybe let's just set m=1 also in the latex? or we write explicit units for the X^2 part </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gcarleo) [inline docs/tutorials/gs_continuous_space.ipynb:1]: <p>can we put the 1/2 in v(x) as well? it's a bit strange to leave it outside I find </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (gcarleo) [inline docs/tutorials/gs_continuous_space.ipynb:1]: <p>why Sigma^-1 = - Identity and not Identity? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (gcarleo) [inline docs/tutorials/gs_continuous_space.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    def minimum_distance(x, sdim, N, L):
</pre><p>was this already defined before? maybe reuse it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (gpescia) [reply docs/tutorials/gs_continuous_space.ipynb:1]: it was to show that you can do things like float * nk.operator, but indeed it does not really fit the tex hamiltonian

File: docs/tutorials/gs-continuous-space.ipynb [hunk 2]
```diff
@@ -0,0 +1,988 @@
+{
```
Comment (gcarleo) [inline docs/tutorials/gs-continuous-space.ipynb:1]: <p>can we try to remove the warning ? </p><p>/home/gabriel/Documents/temp/venv/lib/python3.10/site-packages/netket/vqs/mc/mc_state/state.py:252: UserWarning: n_samples=1000 (1000 per MPI rank) does not divide n_chains=16, increased to 1008 (1008 per MPI rank) </p><p>  self.n_samples = n_samples </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

**Review by PhilipVinc — APPROVED** [review]

**Comment by gcarleo** [comment]
It doesn't seem to appear in the tutorials on the webpage, maybe we missed linking it somewhere? 

image

**Comment by gcarleo** [comment]
now it works thank you @PhilipVinc ! however there is a problem with the [rendering of formulas](https://netket.readthedocs.io/en/latest/tutorials/gs-continuous-space.html), that somehow was not clear from the preview, @gpescia can you see what the issue is? (also, for consistency wit the other tutorials, the author's name should be smaller, see [here ](https://netket.readthedocs.io/en/latest/tutorials/gs-heisenberg.html)) Ah maybe since we are here we could rename it to something shorter and in line with the others: "Continous Space Ground-State Search for Bosonic systems", thanks a lot!

**Pull request merged**